sample-library-depth-a1/build.gradle

```diff
@@ -0,0 +1,16 @@
+apply plugin: 'com.android.library'
```

Comment:
Did you mean to commit these samples here?

Reply:
yep, looks like this was some other work that snuck in.